pkgs/applications/networking/remote/rustdesk/default.nix

```diff
@@ -0,0 +1,113 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeDesktopItem
+, copyDesktopItems
+, makeWrapper
+, rustPlatform
+, cmake
+, yasm
+, nasm
+, zip
+, pkg-config
+, clang
+, gtk3
+, xdotool
+, libxcb
+, libXfixes
+, alsa-lib
+, pulseaudio
+, libXtst
+, libvpx
+, libyuv
+, libopus
+, libsciter
+, llvmPackages
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rustdesk";
+  version = "1.1.8";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
```

Comment:
For readability, I'd recommand not to use `pname` here but instead just typing it out. That way we also don't unnecessarily depend on `pname`, just a small bit more flexible